projects/plugins/social/src/js/store/selectors/jetpack-settings.js

```diff
@@ -2,6 +2,7 @@ const jetpackSettingSelectors = {
 	getPublicizeModuleStatus: state => state.jetpackSettings,
 	isModuleEnabled: state => state.jetpackSettings.publicize_active,
 	isUpdatingJetpackSettings: state => state.jetpackSettings.is_updating,
+	isPremiumPlan: state => ! state.showNudge,
```

Comment:
What do you think about renaming this to `hasPaidPlan`? Premium might indicate a specific plan in the future.

Reply:
Makes sense